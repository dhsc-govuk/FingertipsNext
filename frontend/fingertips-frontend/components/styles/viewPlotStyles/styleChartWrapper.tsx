import styled from 'styled-components';

export const StyleChartWrapper = styled('div')({
  paddingBottom: '50px',
});
