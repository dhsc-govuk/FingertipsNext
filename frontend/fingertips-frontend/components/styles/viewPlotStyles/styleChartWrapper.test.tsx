import { render } from '@testing-library/react';
import { StyleChartWrapper } from './styleChartWrapper';

describe('styleChartWrapper', () => {
  it('should style the ChartWrapper component as expected', () => {
    const container = render(
      <StyleChartWrapper>
        <div data-testid="mock component"></div>
      </StyleChartWrapper>
    );
    expect(container.asFragment()).toMatchSnapshot();
  });
});
