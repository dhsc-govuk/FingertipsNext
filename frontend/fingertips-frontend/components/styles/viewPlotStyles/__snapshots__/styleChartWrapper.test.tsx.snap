// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styleChartWrapper should style the ChartWrapper component as expected 1`] = `
<DocumentFragment>
  .c0 {
  padding-bottom: 50px;
}

<div
    class="c0"
  >
    <div
      data-testid="mock component"
    />
  </div>
</DocumentFragment>
`;
